packages/apm/src/integrations/tracing.ts

```diff
@@ -866,6 +876,25 @@ export class Tracing implements Integration {
       }, timeout);
     }
   }
+
+  /**
+   * Get span based on activity id
+   */
+  public static getActivitySpan(id: number): Span | undefined {
+    if (!id) {
+      return undefined;
+    }
+    if (Tracing._getCurrentHub) {
+      const hub = Tracing._getCurrentHub();
+      if (hub) {
```

Comment:
Why do we need the hub here? Isn't `Tracing._activities[id]` enough?

Reply:
Based off the logic from here https://github.com/getsentry/sentry-javascript/blob/master/packages/apm/src/integrations/tracing.ts#L787-L795

I think you're right though, checking activities is enough.